addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -166,19 +387,646 @@ var Wysiwyg = Widget.extend({
      * @returns {String}
      */
     setValue: function (value, options) {
-        if (this.$editor.is('textarea')) {
-            this.$target.val(value);
+        if (this.$editable.is('textarea')) {
+            this.$editable.val(value);
+        } else {
+            this.$editable.html(value);
+        }
+        this.$editable.html(value);
```

Comment:
why is this called twice? (first time inside the if/else above)
if it's intended, a comment might make it less confusing to the reader ;)

Reply:
After investigation, it is a relic of a past mechanic that was wrongly adapted.
We do not use textarea anymore and the code has been adapted and only use one `this.$editable.html(value);`.